dev-libs/xalan-c/xalan-c-9999.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
It's up to you, but I don't personally believe in live ebuilds without a "template" (if [[ ${PV} == ... ]], etc) because it's really easy for them to get out of sync.

Reply:
I synced 1.12 and 9999 in a separate commit, so I can squash it or drop it easily.  Does it look ok?

Reply:
looks good!

Reply:
Thanks!  I'll squash the commits then